config.json

```diff
@@ -0,0 +1,6 @@
+{
+	"StockAPIURL": "https://api.iextrading.com/1.0/stock/",
```

Comment:
im fine with endpoints being hardcoded but the error messages are more fun configurable

https://12factor.net/config